src/IQFeed.CSharpApiClient/Extensions/DictionaryExtensions.cs

```diff
@@ -0,0 +1,63 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace IQFeed.CSharpApiClient.Extensions
+{
+    public static class DictionaryExtensions
+    {
+        public static string ToString<TK, TV>(this IDictionary<TK, TV> dictionary, string separator)
+        {
+            return ToString(dictionary, (key, value) => $"{key.ToString()}: {value.ToString()}", separator);
+        }
+
+        public static string ToString<TK, TV>(this IDictionary<TK, TV> dictionary, Func<TK, TV, string> formatFunc, string seperator)
+        {
+            var output = new StringBuilder();
+            foreach (var keyValuePair in dictionary)
+            {
+                if (output.Length > 0)
+                    output.Append(seperator);
```

Comment:
I'm a proponent of wrapping brackets around one liners... @mathpaquette thoughts on style here?

Reply:
Just gonna make this the placeholder for this in the rest of the file.